.github/workflows/publish.yaml

```diff
@@ -14,24 +14,24 @@ jobs:
       contents: write
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Create Release
         id: create_release
-        uses: actions/create-release@v1
+        uses: softprops/action-gh-release@v2
```

Comment:
Let's change the CI stuff in another PR. Have we tried this new action somewhere?

Reply:
not yet, will create a separate pr and test it

Reply:
reverted